src/components/SearchResult/SearchResult.jsx

```diff
@@ -1,40 +1,81 @@
+import React, { useState } from 'react';
 import "./SearchResult.scss";
 import FakeBookings from "@/data/fakeBookings.json";
 import TableBody from "./TableBody";
 import TableHead from "./TableHead";
 import dayjs from "dayjs";
 
 const Bookings = FakeBookings;
+
 function SearchResult() {
+  const [sortConfig, setSortConfig] = useState({ key: null, direction: 'ascending' });
+
+  // Function to handle sorting when a column header is clicked
+  const handleSort = (key) => {
+    let direction = 'ascending';
+    if (sortConfig.key === key && sortConfig.direction === 'ascending') {
+      direction = 'descending';
+    }
+    setSortConfig({ key, direction });
+  };
+  
+
+  // Sorting function based on the current sort configuration
+  const sortedBookings = [...Bookings].sort((a, b) => {
```

Comment:
You should be able to do Bookings.sort here. Any particular reason you're done it this way?